src/app/components/util/internal-scroll-button.jsx

```diff
@@ -3,7 +3,8 @@ import {
   Button,
   makeStyles,
 } from '@material-ui/core';
-import { node, string } from 'prop-types';
+import PropTypes, { node, string } from 'prop-types';
```

Comment:
We don't import the entire class in almost all cases. It's generally better to be specific about the imports, both for clarity about what's actually being used, and for file size purposes

```suggestion
import { bool, node, string } from 'prop-types';
```

Reply:
when i tired your code suggestions I was getting a prop types error in multiple locations so I reverted back. We should talk if you have time